airflow/config_templates/config.yml

```diff
@@ -323,6 +323,15 @@
       type: string
       example: ~
       default: "True"
+    - name: store_code
+      description: |
+        Whether to persist DAG files code in DB.
+        If set to True, Webserver reads file contents from DB instead of
+        trying to access files in a DAG folder.
```

Comment:
I don't think this should be a separate config setting. I would say if we have serialized_dags on then this is on. No option.

Reply:
Some users have the DAG File provided by CI/CD. In this case, they want to use the version from the local disk. They do not need a copy in the database, because access to the version from the disk is faster and generates less load.

Reply:
Actually, that is how I started. But then @mik-laj pointed out that it may have huge performance impact. I believe it is safer to hide it behind a feature flag now, so that users can opt to turn it off in case there are problems with it.

Reply:
Can we change it to `store_dag_code`?

Reply:
Sure